src/utils/request-router.ts

```diff
@@ -17,6 +17,8 @@ export enum RequestRouterRegion {
 export type RequestRouterTarget = 'api' | 'ui' | 'assets'
 
 const ingestionDomain = 'i.posthog.com'
+const flagsPath = '/decide'
+const libPath = '/static/array'
```

Comment:
This makes no sense. _this code_ lives in `array.js`. It's only the snippet that loads array.js (the snippet has no idea of any of this code).

This is why I'm generally not in favour of this approach - the snippet is always loaded via the standard endpoint. If that gets blocked then everything is blocked anways.

If you revert the arrayjs stuff then it is more mergeable and at least is an option for people avoiding the snippet (i.e. npm install) but even then I still think this is incredibly unlikely to help...

Reply:
Oh hmmmmm okay. Sorry I have no idea how posthog-js works but I'm learning... I'll think about it.

Reply:
Thoughts on having the snippet itself use this config to get array.js?

Reply:
I think a very small change to the snippet will make it use the `flags_api_host` instead of the standard `api_host` - if the flags host is provided. Basically this part of the snippet:

```
p.src=(s.flags_api_host?s.flags_api_host:s.api_host)+"/static/array.js"
```

(it's somewhat verbose as the [Nullish coalescing operator (`??`)](https://caniuse.com/mdn-javascript_operators_nullish_coalescing) is not IE11 compatible. (when do we stop supporting IE11???????)

I think that's what would fetch array.js initially, and if it's a non-blocked host then it'll get through. 

I think we do still want to do this. If someone wants to use our feature previews, for instance, but while using a tracking/ad blocker, then they can't.